gpii/configs/gpii.config.cloudBased.flowManager.production.json

```diff
@@ -5,7 +5,7 @@
         "authDBServerPort": 5984,
         "distributeOptions": {
             "preferencesDataSource.production.setUrl": {
-                "record": "https://preferences.gpii.net/preferences/%userToken",
+                "record": "@expand:kettle.resolvers.env(GPII_FLOWMANAGER_PREFERENCES_URL, https://preferences.gpii.net/preferences/%userToken)",
```

Comment:
The value you've chosen for the default, https://preferences.gpii.net/preferences/%userToken ,  is already the default value which is set in gpii/node_modules/flowManager/configs/gpii.flowManager.config.production.json which is a base config for this one. So you can remove the default value here and also the specific defaulting mechanism for the env resolver is unneeded. If it isn't working as expected, we should diagnose why.